### PR TITLE
Get the book

### PR DESCRIPTION
**Pull request by nermin**
Per email discussion.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,28 +203,43 @@
...
+					  <h2>Get the book!</h2>
+					  <a href="http://www.artima.com/shop/scala_puzzlers">
+						  <img src="img/scala-puzzlers-book.jpeg" height="220" width="166" alt="Scala Puzzlers book">
+					  </a>
+                	  <p>
+						  for detailed explanations, discussion and workarounds
+					  </p>					  
+                  </div>				  
+				</div>
+			  </div><!--/headline-->
```
Comment (demobox) [inline index.html]: [minor] Indenting here?

**Comment by demobox** [comment]
Thanks! Do you have a screenshot of the rendered result which you could attach..?

**Comment by nermin** [comment]
image

**Comment by demobox** [comment]
Thanks for the screenshot. Nice! Just a couple of comments:

* Do you think it would be easy to adjust the spacing so that the "Get the book" block is the same height as the other one at fullscreen? Something like this:

image

* How about changing the text to "In-depth explanations, discussion, workarounds and more!" or so?

**Comment by demobox** [comment]
Full-screen screenshots of how 653d8b5 renders for me:

Chrome:
image

Firefox:
image

IE:
image

**Comment by nermin** [comment]
> Do you think it would be easy to adjust the spacing so that the "Get the book" block is the same height > as the other one at fullscreen?

I did try, but couldn't get to line up perfectly. Then, I also realized that the height of those boxes depends on the browser window size (for example, try resizing your browser to half size). What do you think about making that yellow color the same as background color?

**Comment by nermin** [comment]
> How about changing the text to "In-depth explanations, discussion, workarounds and more!" or so?
I like that.

**Comment by demobox** [comment]
>  Then, I also realized that the height of those boxes depends on the browser window size (for 
> example, try resizing your browser to half size)

That's actually what I see normally ;-) But I think it's fair enough to assume that most users will have window sizes/screen resolutions where the left-hand box does *not* wrap, so if it lines up in fullscreen view, I'd say that's OK

What does the current version look like in your browsers if the browser window is fullscreen?

**Comment by demobox** [comment]
@nermin: Ping..? ;-)

**Comment by nermin** [comment]
The screenshot I have attached is with the browser fullscreen.

**Comment by demobox** [comment]
> The screenshot I have attached is with the browser fullscreen.

That was the original PR, I guess...could you try again after a `git pull`?

**Comment by nermin** [comment]
Sure, here is full size:
image

**Comment by nermin** [comment]
Not full size:
image

**Comment by nermin** [comment]
I think we should get rid of that darker yellow color.

**Comment by demobox** [comment]
> Sure, here is full size:

Rats ;-) Let's try without the background, indeed.

**Comment by demobox** [comment]
Try now:

image

Certainly better than the two unequal boxes, but I'd prefer this if we could implement it:

image

What do you think?

PS: If we're OK with the current version (i.e. without a background) I'd suggest we commit it just to have something up there. We can always improve later...

**Comment by demobox** [comment]
Squashed what we have so far and merged to master as 2bcd997. We can always revisit if desired.

Thanks, @nermin!

**Comment by nermin** [comment]
Thank you Andrew. The site is overdue for redesign anyway.

**Pull request closed without merging**